assets/svg/index.tsx

```diff
@@ -0,0 +1,18 @@
+export const Bars = () => {
```

Comment:
please can you use a descriptive name for this file

Or even better separate  them into two components ie bars.tsx and times.tsx files

Reply:
so for icons stuff like this its cool to have it exported and mutliple in one place. there wont be issues at all with that method.

Reply:
> so for icons stuff like this its cool to have it exported and mutliple in one place. there wont be issues at all with that method.

yes sire, that way it makes the component more reusable..